### PR TITLE
Ryan story

### PR DESCRIPTION
**Pull request by RyanYoung25**
Please accept my code

### CODE REVIEW

File: PathPlanning.ez [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+%
+%
+%Rotations:
+%   0
+% 3   1
+%   2
+%A = 
+%   rotateRight(robot)
+%   rotateLeft(robot)
+%   moveForward(robot)
```
Comment (jonathanmcdaniel) [inline PathPlanning.ez]: pretty sure im gonna stay right here

File: PathPlanning.ez [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+%
+%D = (In a short hand, informal notation with variables)
+%   rotateRight(robot) causes  at(robot, X, Y, 0) if at(robot, X, Y, 3)
+%   rotateRight(robot) causes  at(robot, X, Y, 1) if at(robot, X, Y, 0)
+%   rotateRight(robot) causes  at(robot, X, Y, 2) if at(robot, X, Y, 1)
+%   rotateRight(robot) causes  at(robot, X, Y, 3) if at(robot, X, Y, 2)
+%
+%   rotateLeft(robot) causes  at(robot, X, Y, 0) if at(robot, X, Y, 1)
+%   rotateLeft(robot) causes  at(robot, X, Y, 1) if at(robot, X, Y, 2)
+%   rotateLeft(robot) causes  at(robot, X, Y, 2) if at(robot, X, Y, 3)
```
Comment (smc395) [inline PathPlanning.ez]: Eew

File: PathPlanning.ez (hunk 1 above)
Comment (RyanYoung25) [inline PathPlanning.ez]: I'm afraid I don't know what that means

File: PathPlanning.ez [hunk 3]
```diff
@@ -91,7 +91,7 @@ holds(F, S+1) :- step(S), fluent(intertial, F), holds(F, S), not -holds(F, S+1).
 
 %Initial State
 holds(at(pioneer, 0, 0, 1), 0).
-%holds(blocked(2,3),0).
+holds(blocked(2,3),0).
```
Comment (aaronmsegal) [inline PathPlanning.ez]: I See....

**Pull request merged**